web/src/components/TaskProgressCheckbox.tsx

```diff
@@ -79,10 +81,8 @@ export const TaskProgressCheckbox = (props: Props): ReactElement => {
     }
 
     if (isTaxTask(props.taskId)) {
-      if (nextStatus === "COMPLETED" && currentOpenModal === undefined) {
-        setCurrentOpenModal("registered-for-taxes");
-        analytics.event.task_status_checkbox.click_completed.show_tax_registration_date_modal();
```

Comment:
Sorry to keep calling these out, but I just saw this event getting removed. Should this event be staying or be modified? I know the behavior is similar, but we may want to either update this or replace it with another event. This could be handled as [Sand], but raising this to the team should happen.

Reply:
Similarly, to what I raised in our chat, I think this should be updated to reflect the current behavior. I'll reach out to the team. TY!